src/routers/peers.py

```
@@ -292,6 +302,136 @@ async def get_peer_card(
+        peer_card = await crud.get_peer_card(
+            db, workspace_id, observer=peer_id, observed=observed
+        )
+
+        return schemas.PeerContext(
+            peer_id=peer_id,
+            target_id=observed,
+            representation=representation,
+            peer_card=peer_card,
+        )
+    except ValueError as e:
+        logger.warning(f"Failed to get context for peer {peer_id}: {str(e)}")
```

Ruff (0.14.7)

Logging statement uses f-string

(G004)